From c8713f39b4375c11238d278b41cfe0711815d323 Mon Sep 17 00:00:00 2001
From: Ivan Vecera <ivecera@redhat.com>
Date: Mon, 8 Sep 2014 16:13:53 -0400
Subject: [ethernet] be2net: include rx-compl error counter in ethtool stats

Message-id: <1410192881-30702-31-git-send-email-ivecera@redhat.com>
Patchwork-id: 92445
O-Subject: [RHEL7.1 PATCH 30/78] be2net: include rx-compl error counter in ethtool stats
Bugzilla: 1088575
RH-Acked-by: Stefan Assmann <sassmann@redhat.com>
RH-Acked-by: Nikolay Aleksandrov <nikolay@redhat.com>

BZ: #1088575

Upstream commit(s):
commit 5c29c09be73755fe99749c99a42c11121d6a8c12
Author: Kalesh AP <kalesh.purayil@emulex.com>
Date:   Fri May 9 13:29:18 2014 +0530

    be2net: include rx-compl error counter in ethtool stats

    Signed-off-by: Kalesh AP <kalesh.purayil@emulex.com>
    Signed-off-by: Sathya Perla <sathya.perla@emulex.com>
    Signed-off-by: David S. Miller <davem@davemloft.net>

Signed-off-by: Ivan Vecera <ivecera@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/net/ethernet/emulex/benet/be_ethtool.c b/drivers/net/ethernet/emulex/benet/be_ethtool.c
index e0dbf63..c1c9233 100644
--- a/drivers/net/ethernet/emulex/benet/be_ethtool.c
+++ b/drivers/net/ethernet/emulex/benet/be_ethtool.c
@@ -132,6 +132,7 @@ static const struct be_ethtool_stat et_rx_stats[] = {
  {DRVSTAT_RX_INFO(rx_bytes)},/* If moving this member see above note */
  {DRVSTAT_RX_INFO(rx_pkts)}, /* If moving this member see above note */
  {DRVSTAT_RX_INFO(rx_compl)},
+ {DRVSTAT_RX_INFO(rx_compl_err)},
  {DRVSTAT_RX_INFO(rx_mcast_pkts)},
  /* Number of page allocation failures while posting receive buffers
   * to HW.
-- 
1.7.1